### PR TITLE
Fix gitignore

### PR DESCRIPTION
**Pull request by samboonzaayer**

### CODE REVIEW

**Review by cmfung — COMMENTED** [review]
@avaboonzaayer Looks good to me, can approve after @boyiq reviews as well!

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,7 @@
+<<<<<<< HEAD
+node_modules
+client/dist/main.js
+=======
 node_modules
 client/dist/main.js
```
Comment (boyiq) [inline .gitignore]: It seems like the merge conflict from main was not correctly resolved, it has both versions right now. Avarice could you please double check? @avaboonzaayer

**Review by boyiq — COMMENTED** [review]
Everything else looks good. Just the gitignore file has 7 lines total including both versions. @avaboonzaayer please double check if the merge conflict is resolved

File: .gitignore (hunk 1 above)
Comment (boyiq) [reply .gitignore]: We might also want to add config.js to the gitignore list. We haven't created the file yet but that's where we will store our github token. Maybe Avarice you could add it while fixing the above issue? @cmfung @avaboonzaayer

Comment (samboonzaayer) [reply .gitignore]: @boyiq I've updated the .gitignore, could you check it again?

**Review by boyiq — APPROVED** [review]

**Pull request merged**